bootstrap/kernel_bootstrap.php

```diff
@@ -25,9 +25,18 @@
 }
 
 $xml = new \SimpleXMLElement(file_get_contents($phpUnitFile));
-$kernelDir = $xml->php[0]->server[0]['value'];
 
-$kernelFile = $rootDir.'/'.$kernelDir.'/AppKernel.php';
+$envDir = $xml->xpath("//php/server[@name='KERNEL_DIR']");
+if (!count($envDir)) {
+    throw new \Exception(
+        'Kernel path must be set via <server name"KERNEL_DIR" value="..."/>'
+    );
+}
+$envClass = $xml->xpath("//php/env[@name='KERNEL_CLASS']");
```

Comment:
Ok. This can be a solution too :-) nice

Reply:
It there is no `<env name="KERNEL_CLASS" ... />` in phpunit.xml.dist, then `$kernelClass` will be equal to `AppKernel` like before my PR

Btw, i made a mistake. I must fix it )

Reply:
Fixed. `$xml->xpath` always returns array, it is empty when no nodes were found